server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -1095,11 +1093,6 @@ exports.registerYourself = function (body, user, originator, xCorrelator, traceI
               forwardingConfigurationInputList
             );
         }
```

Comment:
Incase , if the requestBody is not provided in the request , then we should include this line from 1098 to 1102 in the else part to feed the correct values to the header parameters. 
So that , in the forwardings in line no 1112 , this will be utilized. 
Kindly please include the removed else condition.

Reply:
True, in that case, there is no need to make any changes in AP.